src/daria/corrections/color/colorcorrection.py

```diff
@@ -65,6 +65,9 @@ class ColorCorrection:
     X-rite color checker has four corner landmarks on the colorchecker. The pixel
     coordinates of these have to be provided as input parameters, starting with the
     corner close to the brown swatch, and continuing in counter-clockwise direction.
+    The color correction is applied prior to curvature correction in
+    the daria.Image initialization, and therefore, the roi should be with respect to
+    the uncorrected image.
```

Comment:
I chose this order once, because curvature correction did crop important parts of a color palette in one example. So it made more sense at that time. Now we changed the curvature correction routine. What do you think? Shall we do it differently? Or shall we instead include finding the ROI for the color correction as first step in the jupyer notebook and make it about setting up the correction routines?

Reply:
That sounds like a good idea!

Reply:
Then we could probably also include the option for the color correction to read the config file and include the roi there.

Reply:
I fully agree, and would like that. Witb the previous version it seemed easier with not including it as json did not like slice. But nested lists (what would be needed now) work, they just have to be converted to an array after reading.

Reply:
I'll fix this today then!